compiler/daml-extension/src/extension.ts

```diff
@@ -97,6 +98,24 @@ function getViewColumnForShowResource(): ViewColumn {
     }
 }
 
+function execVisual() {
+        let cmd = "daml clean && daml build && daml damlc visual .daml/dist/*dar > visual.dot"
```

Comment:
Have you tested this on Windows? I am not sure that combining commands with `&&` works there.

Reply:
I have not :(. I am hoping that node cp.exec handles it. Will test it now

Reply:
On windows it does not work :( Will have to look at alternatives

Reply:
You could just execute the individual command separately instead if chaining them with &&. The stdout redirection probably still won't work but you can just capture stdout in a variable like I suggested

Reply:
Broke this up to multiple commands and works on windows too. About the TMP file, I am having an issue with getting the dependency and shipped with vsix

Reply:
No more tmp files.  now all of the tmp files will be cleaned.